framework/src/utils/lib/application-stage.ts

```diff
@@ -57,6 +59,13 @@ export interface ApplicationStageProps extends StageProps {
    * @default - No stage is passed to the application stack
    */
   readonly stage: CICDStage;
+
+  readonly attachIntegrationTest?: boolean;
+
+  /**
+   * Optional parameters that are passed down
```

Comment:
Can you explain a little bit more what it is? I understand it's the parameters that will be passed to the integration tests Stack?